spec/integration/account_deletion_spec.rb

```diff
@@ -1,19 +1,21 @@
 describe "deleteing account", type: :request do
   def account_removal_method
-    AccountDeleter.new(subject.diaspora_handle).perform!
+    AccountDeleter.new(subject).perform!
     subject.reload
   end
 
   context "of local user" do
-    subject(:user) { FactoryGirl.create(:user_with_aspect) }
```

Comment:
I'd rahter not change this part here since it will be hard to rebase. Maybe it's better to add this change on top of my PR instead?